packages/spacebars-compiler/codegen.js

```diff
@@ -134,12 +135,12 @@ Object.assign(CodeGen.prototype, {
             dataCode = 'function () { return { _sequence: ' +
               self.codeGenInclusionData(args.slice(2)) +
               ', _variable: ' + BlazeTools.toJSLiteral(variable) + ' }; }';
-          } else if (path[0] === 'let') {
+          } else if (path[0] === 'let' || path[0] === 'letAwait') {
             var dataProps = {};
             args.forEach(function (arg) {
               if (arg.length !== 3) {
                 // not a keyword arg (x=y)
-                throw new Error("Incorrect form of #let");
+                throw new Error("Incorrect form of " + path[0]);
```

Comment:
Can we improve this error message early on? I'd rather have some more characters in the bundle but a clear error message that helps me in debugging when users report them.

Reply:
Okay my bad I just saw that all these views have these short error messages. We may discuss this separately.

Reply:
Yeah, all of them do that - I'd keep it as a separate topic.